framework/web/js/source/jquery.ba-bbq.js

```diff
@@ -465,7 +465,9 @@
   //  (Object) An object representing the deserialized params string.
   
   $.deparam = jq_deparam = function( params, coerce ) {
```

Comment:
Perhaps you can add a comment header block in the first lines of this file, stating the changes made compared to 1.4pre. Something along the lines of:

```
Additional changes to this file:

- Removed support for IE8 and below
- Fixed prototype pollution (CVE-2021-20086)
```